build/imports-generator.js

```diff
@@ -0,0 +1,89 @@
+/* global Promise */
+
+const fs = require('fs');
+const path = require('path');
+const browserify = require('browserify');
+const { dirname: getDirName } = require('path');
+const makeDir = require('make-dir');
+
+const importsFile = path.join(
+	__dirname,
+	'..',
+	'lib',
+	'core',
+	'imports',
+	'index.js'
+);
+
+// Mapping of external dependencies to be imported
+// Global Namespace : Library name
+const moduleMap = {
+	axios: 'axios',
+	doT: 'dot',
+	CssSelectorParser: 'css-selector-parser'
+};
+
+async function run() {
+	// generate `axe.imports` content
+	const fileContent = `
+	/**
+	* Namespace for imports which holds globals of external module.
+	* @namespace imports
+	* @memberof axe
+	*/
+	axe.imports = {
+		${Object.keys(moduleMap)
+			.reduce((out, key) => {
+				out.push(`'${key}': require('${moduleMap[key]}')`);
+				return out;
+			}, [])
+			.join(', ')}
+	};`;
+
+	// write file content to `lib/core/imports/index.js`
+	try {
+		await createFile(importsFile, fileContent);
+	} catch (error) {
+		throw new Error(
+			`Cannot create file axe.imports with require modules`,
+			error
+		);
+	}
+
+	// browserify `lib/core/imports/index.js`
+	browserify(importsFile).bundle(async (err, result) => {
+		if (err) {
+			throw new Error('Cannot browserify axe.imports', err);
```

Comment:
I'm not sure it makes sense to use "browserify" as a verb 🤔

Reply:
Any thoughts here?

Reply:
No change here. Leaving as is.